packages/@okta/i18n/src/properties/login.properties

```diff
@@ -801,6 +801,16 @@ oie.webauthn.error.not.supported = Security key or biometric authenticator is no
 oie.verify.webauth.title = Verify with Security Key or Biometric Authenticator
 oie.verify.webauthn.instructions = You will be prompted to use a security key or biometric verification (Windows Hello, Touch ID, etc.). Follow the instructions to complete verification.
 oie.verify.webauthn.uv.required.instructions = Biometric verification or a PIN is required to sign in with this authenticator.
+oie.verify.webauthn.cant.verify = Can't verify?
+oie.verify.webauthn.cant.verify.biometric.authenticator.title = Are you trying to use a biometric authenticator?
+oie.verify.webauthn.cant.verify.biometric.authenticator.description1 = Biometric authenticators (fingerprint, face recognition, PIN) will only work on the same device on which they were set up.
+oie.verify.webauthn.cant.verify.biometric.authenticator.description2 = If available, set up another security method that supports verification from a mobile browser.
```

Comment:
nit: a mobile browser

Should it say something like "If available, set up another security method on the device where biometric enrollment was done."

Reply:
@panchamyadav-okta thoughts?

Reply:
@magizh-okta @jasleennuno-okta 
Thanks Magizh, I'd change that slightly to: `If available, set up another security method on the device you used to set up your biometric authenticator.`